src/io/flutter/editor/WidgetIndentsHighlightingPassFactory.java

```diff
@@ -107,135 +86,57 @@ public void caretPositionChanged(@NotNull CaretEvent event) {
       }
     }, this);
 
-    updateActiveEditors();
-    project.getMessageBus().connect(project).subscribe(FileEditorManagerListener.FILE_EDITOR_MANAGER, new FileEditorManagerListener() {
-      public void selectionChanged(@NotNull FileEditorManagerEvent event) {
-        updateActiveEditors();
-      }
-    });
+    editorOutlineService.addListener(outlineListener);
   }
 
-  List<EditorEx> getActiveDartEditors() {
-    if (project.isDisposed()) {
-      return Collections.emptyList();
-    }
-    final FileEditor[] editors = FileEditorManager.getInstance(project).getSelectedEditors();
-    final List<EditorEx> dartEditors = new ArrayList<>();
-    for (FileEditor fileEditor : editors) {
-      if (!(fileEditor instanceof TextEditor)) continue;
-      final TextEditor textEditor = (TextEditor)fileEditor;
-      final Editor editor = textEditor.getEditor();
-      if (editor instanceof EditorEx) {
-        dartEditors.add((EditorEx)editor);
-      }
+  /**
+   * Updates all editors if the settings have changed.
+   *
+   * <p>
+   * This is useful for adding the guides in after they were turned on from the settings menu.
+   */
+  private void syncSettings(FlutterSettings settings) {
+    if (isShowBuildMethodGuides != settings.isShowBuildMethodGuides()) {
+      isShowBuildMethodGuides = settings.isShowBuildMethodGuides();
+      updateAllEditors();
     }
-    return dartEditors;
+    isShowMultipleChildrenGuides = settings.isShowMultipleChildrenGuides() && isShowBuildMethodGuides;
   }
 
-  private void clearListeners() {
-    synchronized (outlineListeners) {
-      final Iterator<Map.Entry<String, FlutterOutlineListener>> entries = outlineListeners.entrySet().iterator();
-
-      while (entries.hasNext()) {
-        final Map.Entry<String, FlutterOutlineListener> entry = entries.next();
-
-        if (entry.getValue() != null) {
-          flutterDartAnalysisService.removeOutlineListener(entry.getKey(), entry.getValue());
+  // Updates the indent guides in a specific editor.
+  private void updateEditors(@NotNull final String path) {
```

Comment:
comment says editor but method name says editors. Unify presumably removing the "s".

Reply:
Done.